packages/riipen-ui/src/components/Grid.jsx

```diff
@@ -45,14 +69,63 @@ class Grid extends React.Component {
     /**
      * Defines the space between grid items in the grid.
      */
-    spacing: PropTypes.oneOf(SPACINGS)
+    spacing: PropTypes.oneOf(SPACINGS),
+
+    /**
+     * The function to callback to when the grid changes sizes
+     * The callback includes the current size of the grid.
+     */
+    sizeChange: PropTypes.func
   };
 
   static defaultProps = {
     alignItems: "flex-start",
     classes: [],
     justifyContent: "flex-start",
-    spacing: 3
+    spacing: 3,
+    flexDirection: "row",
+    flexWrap: "wrap",
+    sizeChange: () => {}
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      size: "lg"
```

Comment:
Is this true? What happens if the screen starts at a mobile resolution?

Reply:
yeah, its really just a fallback for the "this should never happen" situations